packages/next/build/babel/plugins/no-anonymous-default-export.ts

```diff
@@ -23,8 +23,11 @@ export default function NoAnonymousDefaultExport({
     visitor: {
       ExportDefaultDeclaration(path) {
         const def = path.node.declaration
+        // @ts-ignore
+        const bodyType = path.node.body?.type
 
         if (
+          bodyType !== 'JSXElement' ||
```

Comment:
I think this will be useful improvement but haven't tested yet against my gasket app.